src/ResourceManager/RecoveryServices/Commands.RecoveryServices/Vault/RemoveAzureRMRecoveryServicesVault.cs

```diff
@@ -44,11 +44,11 @@ public override void ExecuteCmdlet()
         {
             try
             {
-                RecoveryServicesOperationStatusResponse response = RecoveryServicesClient.DeleteVault(this.Vault.ResourceGroupName, this.Vault.Name);
+                var response = RecoveryServicesClient.DeleteVault(this.Vault.ResourceGroupName, this.Vault.Name);
```

Comment:
Also, cmdlet should support ShouldProcess, and if you do output something, it should be identified in the OutputType attribute